docs/UserGuide.md

```diff
@@ -284,7 +286,18 @@ Format: `focus INDEX`
 * To switch to another Candidate's information, user will just need to type the new command and the new index of the candidate.
 
 ## Scheduling interviews
-
+**:information_source: Note:** 
+
+* The duration of an interview is fixed at 30 minutes. A new feature to allow the user to schedule interviews of flexible duration is
+under consideration for future versions of TAlent Assistant™.
+* Interviews that have expired will automatically be deleted from the interview schedule. An interview is considered
+to be expired once its duration has fully passed. (e.g. If an interview is scheduled at 5PM, the interview is deemed to have expired from 5:31PM onwards).
+If the application is open during the transition of an interview from ongoing to expired, a manual refresh (exit and re-open)
```

Comment:
This line seems to repeat with lines 299-300, maybe we can subsume the whole manual refreshing functionality under line 293? - and also mention that once an interview has expired in the system, the user may also manually delete it to trigger the change from `scheduled` to `completed`.

Reply:
Hey @charmainehly ! Just tested and apparently deleting it manually does not change it to completed. Needs to be a manual restart of the application -> On that note, I think there's a bug -> The interview only gets removed 31 minutes after instead of 30 (e.g. 5.31PM instead of 5.30PM). And yes, it is a typo in my UG I realised.